specification/source/requirements.rst

```diff
@@ -610,48 +610,112 @@ Documentation Export
 
 The documentation export encompasses three items.
 
-#. Engineering schematics of the equipment including the controls points
+#. Engineering schematic of the system including all control points
 
-#. Control points list
+#. Control point list
 
 #. Control sequence description
 
 The composition level at which the functionality will typically be used is the same as the one considered for the configuration widget, for instance primary plant, air handling unit, terminal unit, etc. No specific mechanism to guard against an export call at different levels is required.
 
-:numref:`screen_schematics_output` provides an example of the documentation to be generated in case of an air handling unit. The documentation export may consist in three different files but must contain all the material described in the following paragraphs.
+:numref:`screen_schematics_output` provides an example of the documentation to be generated in case of an air handling unit. The documentation may be exported as three different files but must contain all the material described in the following paragraphs.
 
 .. figure:: img/screen_schematics_output.*
    :name: screen_schematics_output
 
    Mockup of the documentation export
 
 
-Engineering Schematics
+Engineering Schematic
 ======================
 
-Objects of the original model to be included in the schematics export must have a declaration annotation providing the SVG file path for the corresponding engineering symbol e.g. ``annotation(__Linkage(symbol_path="value of symbol_path"))``.
+SVG is the required output format, DXF or DWG is optional.
+
+It is expected that Linkage will eventually be used to generate design documents included in the invitation to tender for HVAC control systems.
+As such, the exported graphics should meet the industry standards and have a pixel perfect precision.
```

Comment:
"pixel perfect" is not clear. Do you mean should be vector graphic? Or that lines need to meet each other?

Reply:
I propose to replace with:

> shape edges, connections between objects, alignment or centering of objects, etc. should be precise at pixel level.

Reply:
That sounds good and is clear.